### DISCUSSION TITLE
New pathway upload

### DISCUSSION THREAD

**Discussion by ondrej-kuda** [wikipathways/wikipathways-help#72] [General]
Hello,
I have created a new pathway and I would like to upload it.
I created the pathway in Pathvisio 4.0.0 and the plugin manager is under construction. The pathway is in GPML/2013a format.
When I log in to Pathvisio 3.3.0, I can start the Upload New pathway routine, but it fails because I do not have write access.
The old way via classic.wikipathways.org does not work anymore.

Could you please suggest how to do this? Thank you.
Best regards,
Ondrej

**Comment by danidi** [comment]
Hello, 
if it is the first time that you want to upload a pathway, your account needs to be given the appropriate permissions. If your wikipathways account has a different name than your github name, can you please tell us? 
@khanspers can you please take a look?
Best regards, 
Daniela

**Reply by ondrej-kuda** [reply]
Hello,
yes, this is the first time. My wikipathways.org account is much older than the github account. 
My account at Wikipathways.org is: Ondrej_Kuda
Thank you for your help,
Best regards,
Ondrej

**Comment by mkutmon** [comment]
I have given you permission to upload! 
I am also curious about your experiences with PV4! How did it go?

**Reply by ondrej-kuda** [reply]
Hi Tina, 
thank you. It works and it is WP5423 from now.
PV400 works better for me, because I can copy-paste multiple objects. This does not work in PV330. 
I wanted to use the PV400 & GPML/2021 format, because I was looking for better formatting options, multiple identifiers, more databases. In the end I did not need it.
Best regards,
Ondrej

**Reply by mkutmon** [reply]
Thanks for your feedback! This is very useful already! We are working hard to get things ready for PV4 and GPML2021 :)
